app/forms/keywords_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class KeywordsForm
+  include ActiveModel::Model
+
+  validates_with KeywordsFormValidator
+
+  attr_accessor :file, :user, :keywords
+
+  def save(params)
+    assign_attributes(params)
+
+    return false if invalid?
+
+    begin
+      if parse_keywords(params[:file])
+        Keyword.create(parse_keywords_from_file(keywords)).map { |keyword| keyword['id'] }
```

Comment:
Can we use `pluck` or shorthand map here?

```suggestion
        Keyword.create(parse_keywords_from_file(keywords)).pluck(:id)
```

```suggestion
        Keyword.create(parse_keywords_from_file(keywords)).map(&:id)
```

Reply:
Didn't know `pluck` was even a thing until now! 😂 Thanks for the suggestion, updated here https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/40f76b1ebffdddf0f3a49ba6f8c7d370bf2afcad